.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
 ignore = E731, T484, T400 # Do not assign a lambda expression, use a def
-max-line-length = 120
\ No newline at end of file
+max-line-length = 121
```

Comment:
needs to be 121 or it disagrees with black.

Reply:
Looks good!